openpype/hosts/maya/plugins/publish/convert_gltf_shader.py

```diff
@@ -0,0 +1,91 @@
+import os
+from maya import cmds
+import pyblish.api
+
+from openpype.pipeline import publish
+
+
+class ConvertGLSLShader(publish.Extractor):
+    """
+    Converting StingrayPBS material to GLSL Shaders
+    specially for the glb export through Maya2GLTF plugin
+    """
```

Comment:
I've brought this up before - but this **should not** be done. A plug-in should not change the local scene and leave it in the changed state.

Publishing should be idempotent so that publishing the scene again would do the same thing as before.

If really required to change the scene for the user and leave the scene changed this should be an explicit repair action on a validator instead.

What if the user were to publish both a Look and a GLTF from the same scene? You'd unexpected results for the look publishing with this plug-in.